DiegoFlores/Day2/Challenge/Person.java

```diff
@@ -0,0 +1,31 @@
+package person;
+
+public class Person {
+
+	private String names;
+	private String surnames;
```

Comment:
surnames? what is sur? improve the class name and use lowerCamelCase example userName